shared/common-adapters/tooltip.desktop.js

```diff
@@ -0,0 +1,61 @@
+// @flow
```

Comment:
We recently added an adapter `toast.desktop.js` that looks pretty similar to this. Could that be used here?

Reply:
Ahhh I had no idea! I searched for "tooltip" and found nothing alike. Will switch to this. Thanks!